subworkflows/nf-core/bedgraph_bedclip_bedgraphtobigwig/meta.yml

```
@@ -0,0 +1,32 @@
+name: "bedgraph_bedclip_bedgraphtobigwig"
+description: Convert gedgraph to bigwig with clip
+keywords:
+  - bedgraph
+  - bigwig
+modules:
+  - ucsc/bedclip
+  - ucsc/bedgraphtobigwig
+input:
+  - bedgraph:
+      type: file
+      description: bedGraph file whoch should be converted
+      pattern: "*.bedGraph"
+  - sizes:
+      type: file
+      description: File with chromosom sizes
+      pattern: "*.sizes"
+output:
+  - bigwig:
+      type: file
+      description: bigWig coverage file relative to genes on the input file
+      pattern: ".bigWig"
+  - bedgraph:
+      type: file
+      description: bedGraph file after clipping
+      pattern: "*.bedGraph"
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+authors:
+  - "@KamilMaliszArdigen"
```

```suggestion
  - "@drpatelh"
  - "@KamilMaliszArdigen"
``` 

I believe @drpatelh wrote it in rnaseq. I think authors is who's responsible for maintaining it.

---

True missed that.